### PR TITLE
Pools page

### PR DESCRIPTION
**Pull request by jsidorenko**

### CODE REVIEW

File: src/components/Pages/Assets/Pools/PoolsView.tsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+          });
+          if (poolInfo[0] === null || poolInfo[1] === null) return <></>;
+
+          return (
+            <SPoolBlock key={pools.lpToken.toPrimitive()}>
+              <SCard activetheme={theme}>
+                <Card.Body>
+                  <SCardEdit className='text-muted'>
+                    <span>Tokens Locked</span>
+                    <Link to='1'>
```
Comment (VadimSaveljev) [inline src/components/Pages/Assets/Pools/PoolsView.tsx:83]: should be some id, and it seems a route is missing to actually read this

File: src/helpers/config.ts [hunk 2]
```diff
@@ -50,3 +53,42 @@ export const extensionConfig = {
...
+                type: 'Option<(Balance,Balance)>',
+              },
+            },
+            version: 1,
+          },
+        ],
+      },
+    },
+  },
+};
```
Comment (VadimSaveljev) [inline src/helpers/config.ts:85]: the things we do to make it work...

File: src/hooks/useAssets.ts [hunk 3]
```diff
@@ -10,14 +10,23 @@ import { useCallback, useState } from 'react';
...
 
-import type { NativeTokenMetadata, TokenBalance, TokenMetadata } from '@helpers/interfaces';
+import type {
+  NativeTokenMetadata,
+  PalletDexPoolInfo,
+  PoolInfo,
+  TokenBalance,
+  TokenMetadata,
+} from '@helpers/interfaces';
+import type { PalletDexPoolId } from '@helpers/interfaces';
```
Comment (VadimSaveljev) [inline src/hooks/useAssets.ts]: looks like this type is a lone wolf
should join the others above though

File: src/components/Pages/Assets/Pools/PoolsView.tsx (hunk 1 above)
Comment (jsidorenko) [reply src/components/Pages/Assets/Pools/PoolsView.tsx:83]: consider this as a placeholder, I'll work on that route in the next PR

File: src/hooks/useAssets.ts (hunk 3 above)
Comment (jsidorenko) [reply src/hooks/useAssets.ts]: it's weird that the prettier doesn't join them automitically

Comment (VadimSaveljev) [reply src/hooks/useAssets.ts]: yep, thought about that, maybe there is a setting hmmm

File: src/components/Pages/Assets/Pools/PoolsView.tsx (hunk 1 above)
Comment (VadimSaveljev) [reply src/components/Pages/Assets/Pools/PoolsView.tsx:83]: I place TODOs in such instances, helps a lot to find things in future

Comment (jsidorenko) [reply src/components/Pages/Assets/Pools/PoolsView.tsx:83]: I didn't place a TODO here as I already started to work on that part in another branch :)

**Review by VadimSaveljev — APPROVED** [review]

**Pull request merged**